### PR TITLE
1120 dowload attachments

### PR DESCRIPTION
**Pull request by VB-at-Bis**
Implements #1120 and #1038

### CODE REVIEW

**Comment by marcellmueller** [comment]
This needs some minor updates to look like the design:
attachments1

attachments

`Export attachments` should be `Download attachments` and placed under the dropdown and the spacing between items is off. 

Figma link:
https://www.figma.com/file/nhOquOVmWD1P3BxEY0KIh5/CCBC-Intake?node-id=9463%3A52282&t=QdcRwW1M2tRcmGFL-0

File: app/pages/analyst/admin/download-attachments.tsx [hunk 1]
```diff
@@ -1,37 +1,125 @@
...
 import { downloadAttachmentsQuery } from '__generated__/downloadAttachmentsQuery.graphql';
 
 const getDownloadAttachmentsQuery = graphql`
   query downloadAttachmentsQuery {
     session {
       sub
-      ...DashboardTabs_query
+      ...DashboardTabs_query  
+    }
+    allIntakes {
```
Comment (marcellmueller) [inline app/pages/analyst/admin/download-attachments.tsx:19]:
```suggestion
    allIntakes(orderBy: CLOSE_TIMESTAMP_DESC) {
```

I think you could simplify this and sort intakes using graphql and remove `sortedNodes` if you wanted to!

Comment (VB-at-Bis) [reply app/pages/analyst/admin/download-attachments.tsx]: Done

**Review by marcellmueller — APPROVED** [review]
Nice job! I'm happy the download attachment is working finally as well, hopefully that is the last of it.

Looks like just needs small eslint fix to merge.

**Pull request merged**